doc/source/overview/dev_practices.rst

```diff
@@ -194,4 +194,372 @@ should not wait until a minor release. These are the steps for a patch release:
 
 Testing
 -------
-<Testing Section>
+Unit testing is critical for the sucessful continious integration and testing of
+any program or libraries belonging to the PyAnsys project are no exception.
+
+There are generally two types of libraries part of the PyAnsys project: those
+that interface or wrap functionality of a different Ansys product, service, or
+application, or tools those that provide functionality. Both types of libraries
+should be tested, but the tests written will depend on purpose of the library.
+For example, a library that is wrapping a gRPC interface would include tests of
+the gRPC methods exposed by the proto files and wrapped by the Python library, but would not be expected to test the functionality of the server.
+
+For example, if testing the gRPC method ``GetNode``:
+
+.. code::
+
+   message Node
+   {
+     int32      id = 1;
+     double     x = 2;
+     double     y = 3;
+     double	z = 4;
+   }
+
+   message NodeRequest {
+     int32		num = 1;
+   }
+
+   message NodeResponse {
+     Node		node = 1;
+   }
+
+  service SomeService{
+
+     rpc GetNode(NodeRequest)				returns (NodeResponse);
+     // other methods
+   }
+
+Then youyour unit test would test the wrapped python function (for example,
+``get_node``).  For example the ``get_node`` method might be implemented as:
+
+.. code:: python
+
+   from ansys.product.service.v0 import service_pb2
+
+   def get_node(self, index):
+       """Return the coordinates of a node for a given index.
+
+       Parameters
+       ----------
+       index : int
+           Index of the node.
+
+       Returns
+       -------
+       tuple
+           Coordinates of the node.
+
+       Examples
+       --------
+       >>> from ansys.product.service import SomeService
+       >>> srv = SomeService()
+       >>> srv.create_node(1, 4.5, 9.0, 3.2)
+       >>> node = srv.get_node(1)
+       >>> node
+       (4.5, 9.0, 3.2)
+
+       """
+       resp = service_pb2.GetNode(index=index)
+       return resp.x, resp.y, resp.z
+
+Your test would be implemented (within ``tests/test_nodes.py``):
```

Comment:
It seems that we can remove the parentheses as this information seems essential.